values-vvp.yaml

```diff
@@ -28,6 +27,16 @@ vvp:
             state.backend: filesystem
             taskmanager.memory.managed.fraction: 0.0 # no managed memory needed for filesystem statebackend
             high-availability: vvp-kubernetes
+            metrics.reporter.prom.class: org.apache.flink.metrics.prometheus.PrometheusReporter
```

Comment:
Do you think it will confuse users that we configure these defaults even if the metrics stack is not installed? Alternatively, we could put these values into its own values file, which is only used during installation if the user passes an additional argument to the ./setup.sh indicating to install the optional metrics components.

Reply:
Yes, we're on the same page about wanting to make this a flag on setup.sh, but my opinion in the PR message was to merge this and then follow up with that improvement.

Without that flag, it's not really that easy to add these defaults post hoc—unless we try for a clever `helm upgrade`. But I think it's best we just add the flag.